nextflow.config

```diff
@@ -61,7 +61,6 @@ params {
     cf_window = null                    // by default we are not using this in Control-FREEC
 
     generate_gvcf = false // g.vcf are not produced by HaplotypeCaller by default
-    no_strelka_bp = false // Strelka will use Manta candidateSmallIndels if available
```

Comment:
what happens to this option?

Reply:
ew said we don't use it anymore at some point but it was still in the config. can add it back to the code base though

Reply:
I'm happy with removing it.
If people don't want to follow bp, they can always run manta/strelka separately

Reply:
👍 sounds good.